src/stores/sessions.js

```diff
@@ -36,6 +40,56 @@ export const useSessionsStore = defineStore('sessions', {
           }
         }
       }
+    },
+    async fetchToken () {
+      const requestBody = JSON.stringify(this.currentSession)
+      const response = await fetchApiCall({
+        url: 'http://rti-bridge-dev.lco.gtn/login',
+        method: 'POST',
+        body: JSON.parse(requestBody)
+      })
+      if (!this.getTokenForCurrentSession) {
+        this.sessionTokens[this.currentSessionId] = response.token
+      }
+    },
+    async fetchStatus () {
+      const token = this.getTokenForCurrentSession
+      if (!token) {
+        await this.fetchToken()
+      }
+
+      const response = await fetchApiCall({
+        url: 'http://rti-bridge-dev.lco.gtn/session_status',
```

Comment:
This URL could go in global config

Reply:
my idea is to have the base urls in the config files and then append `+/session_status` or whatever ending it is. Let me know if you think adding each url makes more sense :)

Reply:
I think that's fine too